test/data_static.jl

```diff
@@ -74,6 +74,7 @@ test_toyModel() = CoreModel(
 )
 
 const reaction_standard_gibbs_free_energies = Dict(
+    # data from Equilibrator using the E. coli core model's reactions
```

Comment:
is there a proper source? did you generate the data? How did you generate the data?

Reply:
no proper source, but I will be a little more thorough